scripts/transparencia_dados_abertos_brasil.json

```diff
@@ -0,0 +1,73 @@
+{
+    "citation": "NA",
```

Comment:
I think we should use a citation like the one in this answer since they don't indicate a preferred citation:
https://academia.stackexchange.com/a/14015